docs/sources/next/javascript-api/k6-experimental/browser/element-handle/dblclick.md

```diff
@@ -0,0 +1,58 @@
+---
+title: 'dblclick([, options])'
+excerpt: 'Browser module: elementHandle.dblclick([, options]) method'
+---
+
+# dblclick([, options])
+
+Mouse double clicks the element.
+
+<TableWithNestedRows>
+
+| Parameter           | Type     | Default | Description                                                                                                                                                                                                                                                                                                                                   |
+| ------------------- | -------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| options             | object   | `null`  |                                                                                                                                                                                                                                                                                                                                               |
+| options.button      | string   | `left`  | The mouse button (`left`, `middle` or `right`) to use during the action.                                                                                                                                                                                                                                                                      |
+| options.delay       | number   | `0`     | Milliseconds to wait between `mousedown` and `mouseup`.                                                                                                                                                                                                                                                                                       |
+| options.force       | boolean  | `false` | Setting this to `true` will bypass the actionability checks (`visible`, `stable`, `enabled`).                                                                                                                                                                                                                                                 |
```

Comment:
Not a strong opinion, but?

```suggestion
| options.force       | boolean  | `false` | Setting this to `true` bypasses the actionability checks (`visible`, `stable`, `enabled`).      
```

Reply:
Resolved in 0abcf69969a3e825f97446543ddf9575c3bf0ff0